### ISSUE TITLE
_dynet.ParameterCollection object has no attribute 'load'

### ISSUE THREAD

**Issue by huangjingxian** [clab/dynet#642]
There is always an issue when I want to use dynet.ParameterCollection.load(filename)

**Comment by zeeeyang** [comment]
Try to use **populate** instead of **load**. See [http://dynet.readthedocs.io/en/latest/cpp_saving_tutorial.html?highlight=load](http://dynet.readthedocs.io/en/latest/cpp_saving_tutorial.html?highlight=load)

**Comment by xunzhang** [comment]
@huangjingxian , the high-level Python interface is `dynet.load` if you want to use. You could also use `ParameterCollection.load_param` and `ParameterCollection.load_lookup_param` interfaces. Please check out usage and examples here https://dynet.readthedocs.io/en/latest/python_saving_tutorial.html .

**Comment by Horsmann** [comment]
I had the same exception but got it working after trying around a bit (based on a version pulled from master `today`). My mistake was that I mixed up API calls.

I mixed up / did wrong:
`dy.save / dy.load()` vs. `model.save() / model.populate()`
Furthermore, I was a bit confused that I had to carry the dimensions of the `_lookup_parameters` to the code that loads the model. I expected that to happen automatically / being part of the serialization.

- `model.save()` creates a file from a filename and adds `.data` as suffix but `model.populate()` does not automatically attach this suffix - you have add the suffix by hand

btw. what is the difference between `dy.Model()` and `dy.ParameterCollection()` seems to be interchangeable
